src/Crowdin.Api/Workflows/WorkflowsApiExecutor.cs

```diff
@@ -51,6 +54,47 @@ public async Task<WorkflowStep> GetWorkflowStep(int projectId, int stepId)
             return _jsonParser.ParseResponseObject<WorkflowStep>(result.JsonObject);
         }
 
+        /// <summary>
```

Comment:
I don't think we need two APIs here, especially for new methods. This is extra maintenance and complexity for the end user. I suggest to keep only the second one (with `StringsOnTheWorkflowStepListParams`)

Reply:
Which one is more consistent for our codebase?

Reply:
Some List methods (especially older ones introduced with the initial V2 release) already include similar overloads. Given that, I believe both overloads align well with the existing codebase and maintain consistency.